packages/clay-drop-down/stories/index.tsx

```diff
@@ -96,22 +96,42 @@ const DropDownWithState: React.FunctionComponent<any> = ({
 };
 
 storiesOf('Components|ClayDropDown', module)
-	.add('default', () => (
-		<DropDownWithState>
-			<ClayDropDown.ItemList>
-				{[
-					{href: '#one', label: 'one'},
-					{href: '#two', label: 'two'},
-					{disabled: true, href: '#three', label: 'three'},
-					{href: '#four', label: 'four'},
-				].map(({href, label, ...otherProps}, i) => (
-					<ClayDropDown.Item href={href} key={i} {...otherProps}>
-						{label}
-					</ClayDropDown.Item>
-				))}
-			</ClayDropDown.ItemList>
-		</DropDownWithState>
-	))
+	.add('default', () => {
+		React.useEffect(() => {
+			document
+				.getElementById('testtest')
+				.append(
+					document
+						.createRange()
+						.createContextualFragment(
+							'<button onclick="alert();">test</button>'
+						)
+				);
+		}, []);
```

Comment:
This is just a hack for storybook to render an element outside of the react tree for testing purposes